src/data/mockDecks.ts

```diff
@@ -0,0 +1,46 @@
+export type Card = {
+    id: string;
+    front: string;
+    back: string;
+  };
+  
+export type Deck = {
+    id: string;
+    name: string;
+    cards: Card[];
+    progress: number;
+  };
+  
+export const mockDecks: Deck[] = [
+    {
+      id: '1',
+      name: 'Mathematics',
+      progress: 75,
+      cards: [
+        {
+          id: '1',
+          front: 'Što je Pitagorin poučak?',
```

Comment:
please change front key to 'Question' and back key to 'Answer' according to backend architecture.